libs/core/src/components/sage-image/stories/sage-image.stories.mdx

```diff
@@ -0,0 +1,61 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+import { html, render } from 'lit-html';
+
+import { extractArgTypes } from '@pxtrn/storybook-addon-docs-stencil';
+
+<Meta title="webcomponents/Image" component="sage-image" argTypes={extractArgTypes('sage-image')} />
+
+export const Template = (args) => html`<sage-image alt="${args.alt}" src="${args.src}" />`;
+export const SizesAndSrcset = (args) => html`<sage-image alt="${args.alt}" sizes="${args.sizes}" src="${args.src}" srcset="${args.srcset}" />`;
+
+# Image
+
+An image can be used to convey supplemental information to the user or as decoration.
+
+## Guidelines
+
+### Accessibility
+
+All images, regardless of use, must have an appropriate alt text to describe the image for screen reader users. If the image only provides visual decoration to the page, be sure to add `alt=""` so that screen readers will properly skip over the element. For more information on how to properly use alt text for images, please refer to the [alt decision tree](https://www.w3.org/WAI/tutorials/images/decision-tree/).
```

Comment:
Should we as the implementors provide a fallback should one not be applied? e.g the name of the file

Reply:
No, and I should actually consider removing or revising that sentence as the `alt` prop already defaults to an empty string. The reason for the no here is because I don't think we should assume that any image added should have context as there will inevitably be situations where the image will have no significance to the context of the page (i.e. it's decoration). Decorative images should always have an empty `alt` in order to be skipped over being read in the accessibility tree for screen reader users. This is putting more of the onus on the engineers to make the correct decision about proper usage, but that's also why I linked a page here that _should_ make that decision easy.